.github/workflows/ci.yml

```diff
@@ -231,6 +232,18 @@ jobs:
           TARGET: ${{ matrix.target }}
           SUB: ${{ matrix.sub }}
         shell: bash
+
+      - name: LLVM instrument coverage
+        id: cov
+        uses: ./.github/actions/cargo-llvm-cov
+        if: steps.prepare-meta.outputs.has-image
+        with:
```

Comment:
Probably change to:

```yaml
if: steps.prepare-meta.outputs.has-image && matrix.target == 'x86_64-unknown-linux-gnu'
```

Reply:
We don't need (or want) coverage on the test binaries, fixed this by clearing RUSTFLAGS, maybe it should be more specific though (i.e only remove `instrument-coverage` but not sure how to do that well)